SU-SWS/react_paragraphs
# Change paths for compiled css

# READY FOR REVIEW

# Summary
- Not sure if you can fix the compiled paths, but this is one solution.
- You modifications to webpack also seem to have brought up a couple of files like `react_paragraphs.react_paragraphs.field_formatter.min.js` and `react_paragraphs.react_paragraphs.field_widget.min.js`. they don't do anything but kinda pointless files.
# Need Review By (Date)
- soon

# Urgency
- low

# Steps to Test
1. checkout this branch
1. `npm install` or `yarn`
1. make a change in `field_formatter.scss`
1. `npm run build` or `yarn build`
1. verify the change applied to the css in `js/build/css` directory.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
